PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -140,9 +124,9 @@ do
 			GestureArea.BackgroundTransparency = 1.0
 			GestureArea.Visible = true
 			GestureArea.BackgroundColor3 = Color3.fromRGB(0, 0, 0)
-			GestureArea.Size = UDim2.new(1, 0, .6, 0)
-			GestureArea.Position = UDim2.new(0, 0, 0, 0)
 			GestureArea.Parent = ScreenGui
+			layoutGestureArea(PortraitMode)
```

Comment:
we should layout before parenting, causes less signals to be fired to rendering. Not a huge deal though.